pkg/resource/resource.go

```diff
@@ -552,3 +552,41 @@ func SetDefaultLabels(r *model.Resource, env *model.Environment) error {
 
 	return nil
 }
+
+// SetCloneDefaultLabels sets default labels for cloned resource.
+func SetCloneDefaultLabels(r *model.Resource, env *model.Environment) error {
```

Comment:
Why don't we make SetDefaultLabels() handle both cases?

Reply:
For user create case in UI, if user intend to fill the `stop` label, we will not remove the `stop` label for user. This is only for clone case.

Reply:
What if user intend to fill the stop label on env clone? Should we remove the labels in the frontend when clone?

Reply:
It maybe better handler this in UI. @hibig

Reply:
Per discussion we keep this. Users need to explicitly update the label to make it stoppable in production.